src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react'
+import { createRoot } from 'react-dom/client'
+import App from './App.tsx'
+import './index.css'
+
```

Comment:
**praise**: Excellent use of `createRoot` from `react-dom/client` for rendering, which is the recommended approach in React 18.